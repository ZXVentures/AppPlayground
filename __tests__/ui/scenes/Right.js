import 'react-native'
import React from 'react' // eslint-disable-line no-unused-vars
import Right from 'ui/scenes/Right'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('ui/scenes/Right', () => {
  it('snapshot test', () => {
    const tree = renderer.create(
      <Right />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
