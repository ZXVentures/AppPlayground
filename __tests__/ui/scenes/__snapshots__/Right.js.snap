exports[`ui/scenes/Right snapshot test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#00b8d4",
      "flex": 1,
      "justifyContent": "center",
    }
  } />
`;
